client/src/components/Sidebar/Bottom.js

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Button,
+  Chip,
+  Divider,
+  Grid,
+  TextField,
+  Typography,
+} from "@material-ui/core";
+import PropTypes from 'prop-types';
+
+const useSpyBottomStyles = makeStyles(theme => ({
+  bottom: {
+    width: "100%",
+    height: "auto",
+    marginTop: theme.spacing(1),
+  },
+  done: {
+    marginTop: theme.spacing(1),
+    display: "block",
+    backgroundColor: "rgb(38, 182, 92)",
+    color: "white",
+    width: "50%",
+  },
+  text: {
+    width: "100%",
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+  },
+  chip: {
+    borderRadius: "8px",
+    display: "inline-block",
+    marginLeft: theme.spacing(0.5),
+    marginRight: theme.spacing(0.5),
+    paddingTop: "3px;",
+    color: "rgb(183, 183, 183);",
+  },
+  count: {
+  },
+  prompt: {
+    backgroundColor: "rgb(75, 75, 75);",
+    color: "white",
+    marginTop: theme.spacing(1),
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(1),
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+    borderRadius: "8px",
+  },
+  triangle: {
+    content: ' ',
+    width: theme.spacing(0),
+    height: theme.spacing(0),
+    border: theme.spacing(0),
+    borderLeft: `${theme.spacing(1)}px solid transparent;`,
+    borderRight: `${theme.spacing(1)}px solid transparent;`,
+    borderTop: `${theme.spacing(1)}px solid rgb(75, 75, 75);`,
+  },
+  hDivider: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    width: "100%",
+    height: "1px;",
+  }
+}));
+
+const SpyBottom = ({ gameID, countMax, setFinished, player, socket}) => {
+  const classes = useSpyBottomStyles();
+  const [spyHint, setSpyHint] = useState("");
+  const [count, setCount] = useState(1);
+  // const onClickDone = () => setFinished(true);
```

Comment:
Not sure if this is your change or from someone else's but if we're not using this code you can remove it